source/beamline/AMPVControl.h

```diff
@@ -114,6 +114,12 @@ class AMReadOnlyPVControl : public AMControl {
 	///  This extra signal is specialized to report on PV channel connection status.  You should be free to ignore it and use the signals defined in AMControl.
 	void readConnectionTimeoutOccurred();
 
+public slots:
+	/// Sets the minimum value.
+	void setMinimumValue(double newValue);
+	/// Sets the maximum value.
+	void setMaximumValue(double newValue);
```

Comment:
Those names are probably clearer, especially as we aren't actually pushing limits to the EPICS layer. There would need to be a distinction between setting the Acquaman/softer-limits and the EPICS/soft-limits, whenever we develop that capability.